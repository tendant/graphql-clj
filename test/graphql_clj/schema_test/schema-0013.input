type Subscription {
  commentAdded(postID: String!): String
}