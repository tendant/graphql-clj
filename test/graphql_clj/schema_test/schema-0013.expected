{:subscriptions
 {:commentAdded
  {:args {:postID {:type (non-null String)}}, :type String}}}
